mlflow/server/js/src/components/ExperimentRunsSortToggle.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default class ExperimentRunsSortToggle extends React.Component {
```

Comment:
Why do we need this class actually?

Reply:
Good question, this class provides a custom dropdown toggle as per the pattern in the [react-bootstrap docs](https://react-bootstrap.github.io/components/dropdowns/#btn-dropdowns-custom) (cmd+f for "Custom Dropdown Components"). We pass `bsRole` to this component as described in the docs. `bsClass` seems to be passed to this component by the react-bootstrap framework, so we include it here to avoid PropType validation errors.